src/components/App/_App.scss

```diff
@@ -1,6 +1,4 @@
-// .App {
-
-// }
+@import '../../variables.scss';
```

Comment:
Do we need this here?? I have a variable file in my PR
I already grabbed many of the colors from the Figma Mock up

Reply:
Well it wasn't merged yet, so this is just a conflict of our two separate branches we'll need to address.